tests/crypto/rdrand_test.c

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#define MAX_LOOP_SIZE 100000000
+#include <openenclave/internal/tests.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include "../../common/common.h"
+
+void TestRdrand()
```

Comment:
Please add a comment that this test is trying to ensure that the `_rdrand` function is incorporates the blocking wait to retry if the `RDRAND` instruction is out of entropy.

Reply:
added